onecodex/cli.py

```diff
@@ -492,6 +492,40 @@ def upload(
     )
 
 
+@onecodex.command("upload_asset")
+@click.option(
+    "--max-threads",
+    default=DEFAULT_THREADS,
+    help=OPTION_HELP["max_threads"],
+    metavar="<int:threads>",
+)
+@click.argument(
+    "file",
+    nargs=1,
+    required=True,
+    type=click.Path(exists=True),
+    shell_complete=partial(click_path_autocomplete_helper, directory=False),
+)
+@click.pass_context
+@pretty_errors
+@telemetry
+@login_required(experimental=True)
```

Comment:
Nitpick: It is not clear what does `experimental` stand for. It seems that it tell to use an experimental login service or something.

Maybe  `@login_required(experimental_api=True)` or even more generic
`@login_required(api_kwargs={"experimental": True})`.